docs/en/building/introduction.md

```diff
@@ -0,0 +1,15 @@
+| -------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Dojo CLI                   | A modular command-line tool useful for kick-starting a new application, creating widgets, and running tests.                                                                                                                                                                                                    |
+| Development server         | A local development web service that watches the file system and automatically rebuilds when changes are detected. Also includes support for https and proxying requests.                                                                                                                                       |
+| Bundles                    | Create a better user experience by reducing how much users need to download and improving TTL. Bundles can be automatically created by route or explicitly defined in configuration.                                                                                                                            |
+| Conditional code inclusion | Features defined via dojo/has can be statically turned off/on through the `.dojorc` configuration file. Code branches that are inaccessible due to these settings are automatically elided. This makes it easy to support features of specific targets like ie11 or mobile without compromising on bundle size. |
+| PWA Support                | Progressive web apps create a faster more reliable user experience by caching content and even working offline. Dojo makes it easy to create a service worker through configuration or define it in code and build it as part of your application.                                                              |
+| Build-time rendering       | Renders routes during build time to generate initial HTML and CSS. By rendering at build time Dojo can skip the cost of producing an initial render and create a more responsive application with little additional complexity.                                                                                 |
+| Ejectable configurations   | Take total control of the build process by ejecting the underlying configurations and work directly with the low-level tools. An escape hatch for anyone who needs complex or specific functionality outside of the norm.                                                                                       |
```

Comment:
I'd say we can keep this feature somewhat buried in the supplemental - we'd prefer if users favoured the framework's build pipeline and fed back if it didn't meet their needs. Ejecting is effectively a last resort.